### ISSUE TITLE
Add file extension as attribute

### ISSUE THREAD

**Issue by tim-hilde** [tadashi-aikawa/obsidian-another-quick-switcher#139]
Please add the file extension as an attribute. I would like to style the entry depending on the extension.

**Comment by tadashi-aikawa** [comment]
Hi, @tim-hilde :)

I think it is better that adds an `ext` attribute if you want to set styles only. If you are OK, I'll implement it.

![image](https://user-images.githubusercontent.com/9500018/212051369-e250dcb1-3fd4-4e63-b086-ca4b73aee631.png)

**Comment by tim-hilde** [comment]
Yes, that looks good. Thank you!

**Comment by tadashi-aikawa** [comment]
I have fixed it. Please wait for the next release.

![image](https://user-images.githubusercontent.com/9500018/212889922-da170394-390c-4b2f-84b5-253670095df8.png)

```css
/* It works */
.another-quick-switcher__item[extension="png"] {
  background-color: red;
}
```

**Comment by tadashi-aikawa** [comment]
@tim-hilde 
Released in [v8.4.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/8.4.0) 🚀
